### PR TITLE
Possible new post on visualizing TL data

### PR DESCRIPTION
**Pull request by kuanb**
Might be a bit meandering... Happy to edit and improve, definitely just wrote without much direction... If you want to see it rendered with images, I stuck it on my blog too, as a first draft: http://kuanbutts.com/2017/10/12/transitland-boxes/

### CODE REVIEW

**Comment by kuanb** [comment]
Just wanted to check in on this per conversation from last week - make sure I'm not holding anything up on my end.

File: _posts/2017-10-25-geopandas-tutorial.md [hunk 1]
```diff
@@ -221,20 +221,22 @@ aus_nz = {  "type": "Polygon",
...
 
-![eurasia](https://raw.githubusercontent.com/kuanb/kuanb.github.io/master/images/_posts/transitland-boxes/eurasia.png)
+![eurasia](/images/geopandas-tutorial/eurasia.png)
 
 Above, another plot, this time of Europe and western Russia. As you can see, this pattern is easily reproducible anywhere.
 
 # Conclusion
 
-At this point you should be free to explore the feeds available on Transit.Land - there's a ton of them! Need ideas for what to explore next? How about choosing a more refined base layer if you'd like to work at a "closer" scale and explore and work with transit bounds data at, for example, the county and census tract scale in the United States.
+At this point you should be free to explore the feeds available on Transitland &mdash; there's a ton of them! Need ideas for what to explore next? How about choosing a more refined base layer if you'd like to work at a "closer" scale and explore and work with transit bounds data at, for example, the county and census tract scale in the United States.
```
Comment (kuanb) [inline _posts/2017-10-25-geopandas-tutorial.md]: If you want, on my blog 9http://kuanbutts.com/2017/10/12/transitland-boxes/) I did add in one more image, here: https://raw.githubusercontent.com/kuanb/kuanb.github.io/master/images/_posts/transitland-boxes/world.png

Similarly, I also generated this image today, should it be of interest (could be added in the US section):
https://pbs.twimg.com/media/DNAU1e0UEAAvgbI.png:large

File: _posts/2017-10-25-geopandas-tutorial.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ def recursive_query(target_area=None, full_url=None):
     
     if target_area:
         bbox = ','.join(map(lambda x: str(x), target_area.bounds))
-        resp = requests.get(url='https://transit.land/api/v1/operators', params={'bbox': bbox, 'per_page': 50})
+        resp = requests.get(url='https://Transitland/api/v1/operators', params={'bbox': bbox, 'per_page': 50})
```
Comment (kuanb) [inline _posts/2017-10-25-geopandas-tutorial.md]: is there some contextual text parsing during the build process that will convert that to transit.land - or a reason why it was switched from transit.land to Transitland?

Comment (drewda) [reply _posts/2017-10-25-geopandas-tutorial.md]: Just some excessive use of multi-cursor text editing.

**Comment by kuanb** [comment]
looks great, thx!

**Pull request merged**